include/ginkgo/core/distributed/preconditioner/schwarz.hpp

```diff
@@ -99,6 +107,32 @@ class Schwarz
          * local solver.
          */
         bool GKO_FACTORY_PARAMETER_SCALAR(l1_smoother, false);
```

Comment:
add a comment about it is unsupported with two level

Reply:
It is supported with two-level. L1 being a local smoother is separate from the coarse solve.